chain/epoch-manager/src/lib.rs

```diff
@@ -1503,26 +1504,26 @@ impl EpochManager {
             ValidatorInfoIdentifier::BlockHash(ref b) => self.get_epoch_id(b)?,
         };
         let cur_epoch_info = self.get_epoch_info(&epoch_id)?;
-        let cur_shard_layout = self.get_shard_layout(&epoch_id)?;
         let epoch_height = cur_epoch_info.epoch_height();
         let epoch_start_height = self.get_epoch_start_from_epoch_id(&epoch_id)?;
-        let mut validator_to_shard = (0..cur_epoch_info.validators_len())
```

Comment:
this block is moved to line 1515 because it is only needed for `ValidatorInfoIdentifier::EpochId` branch. The other branch uses the aggregator stats.